microsoft/testsuites/vm_extensions/waagent.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import uuid
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.sut_orchestrator.azure.features import AzureExtension
+
+
+@TestSuiteMetadata(
+    area="azure",
+    category="functional",
+    description="BVT for VM Agent",
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class WaAgentBvt(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        Runs an extension and verifies it executed on the remote machine.
```

Comment:
Run CustomScript extension

Reply:
No, the extension is irrelevant. This is a test for the agent, not for Custom Script

Reply:
Although the test is not related to custom script, the test approach is to use the custom script. The description is used to generate test spec, so it needs to explain what's tested, and how it's tested.  More details can save others' time to look the code.

Reply:
done